.cirrus.yml

```diff
@@ -624,3 +624,40 @@ cluster_testing_docker_builder:
       path: "testing/external/zeek-testing-cluster/.tmp/**"
   depends_on:
     - amd64_container_image
+
+
+# Test zeekctl upon master and release pushes and also when
+# a PR has a zeekctlci or fullci label.
+#
+# Also triggers on CIRRUS_CRON == 'zeekctl-ngihtly' if that is configured
+# through the Cirrus Web UI.
+zeekctl_debian11_task:
+  cpu: *CPUS
+  memory: *MEMORY
+  only_if: >
+    ( $CIRRUS_CRON == 'zeekctl-nightly' ) ||
+    ( $CIRRUS_PR != '' && $CIRRUS_PR_LABELS =~ '.*(zeekctlci|fullci).*' ) ||
+    ( $CIRRUS_REPO_NAME == 'zeek' && (
+        $CIRRUS_BRANCH == 'master' ||
+        $CIRRUS_BRANCH =~ 'release/.*' )
+    )
+  container:
+    # Debian 11 EOL: June 2026
+    dockerfile: ci/debian-11/Dockerfile
+    << : *RESOURCES_TEMPLATE
+  sync_submodules_script: git submodule update --recursive --init
+  always:
+    ccache_cache:
```

Comment:
What needs to be cached in this build?

Reply:
The zeekctl testing setup builds Zeek from scratch (from the top level):

https://github.com/zeek/zeekctl/blob/266890af6e78c16b64734c68b55147d17b3a019b/testing/Scripts/build-zeek#L3-L5

Basically the `Scripts/build-zeek` script below. This happens when you run `make` within `testing` and I didn't want to re-structure things a lot given this is mostly to ascertain zeectl doesn't go fully out-of date.